plugins/inlinetoolbar/plugin.js

```diff
@@ -99,6 +135,62 @@
 					that.attach( element, false );
 				} ) );
 			};
+
+			/**
+			 * Render items and attach view to DOM element.
```

Comment:
This doc is not enough. `create` method is extremely important (key to create), thus docs need to be good.

`InlineToolbarView.create` method has much better description, but I'd avoid raw copy'n'pasting.